build_tools/packaging/python/templates/rocm-sdk-core/setup.py

```diff
@@ -74,6 +74,16 @@ def import_dist_info():
                 "rocminfo=rocm_sdk_core._cli:rocm_info",
                 "rocm-smi=rocm_sdk_core._cli:rocm_smi",
             ]
+            + (
+                [
+                    # These profiler tools are only available on Linux if built.
+                    "rocprofv3=rocm_sdk_core._cli:rocprofv3",
+                    "rocprofv3-attach=rocm_sdk_core._cli:rocprofv3_attach",
+                    "rocprofv3-avail=rocm_sdk_core._cli:rocprofv3_avail",
+                ]
+                if (Path("platform") / platform_package_name / "bin" / "rocprofv3").exists()
+                else []
```

Comment:
Not sure we need this if clause here. Packaging so far assumes that all components get build. Technically you can skip other components as well but within the templates we do not check if those are present.